### PR TITLE
[jsk_apc2016_common] apc2016 object name fixed and add test 

### LINKED ISSUE #1431: Name of objects for APC2016?
Are you aware about this? @knorth55

### PR DESCRIPTION
**Pull request by knorth55**
Closes #1431

### CODE REVIEW

**Comment by wkentaro** [comment]
According to this: https://raw.githubusercontent.com/wkentaro/jsk_apc/37d93c602982f1f1f3dd3a4db93446ca6fdd1bba/doc/_images/apc2016_bonus_and_stocks.jpg

The possible number of objects are fixed. (`2x` means it has 2 stocks) So please update the `stock` entry also.

You need to add some test here:
https://github.com/start-jsk/jsk_apc/blob/master/jsk_apc2016_common%2Fpython%2Fjsk_apc2016_common%2Ftests%2F__init__.py

maybe for 
- number of objects
- whether entries are sorted in alphabetical order
- expected entries (`name`, `weight`, `graspability`, `stock`)

**Comment by knorth55** [comment]
i checked and i thought stock entry is already correct and no need to modify

**Comment by knorth55** [comment]
@wkentaro what do you mean alphabetical sort for entries? You mean alphabetical sort for object names?

**Comment by wkentaro** [comment]
> i checked and i thought stock entry is already correct and no need to modify

Oh, I see. thanks.

**Comment by wkentaro** [comment]
> what do you mean alphabetical sort for entries? You mean alphabetical sort for object names?

Yes.

File: jsk_apc2016_common/python/jsk_apc2016_common/__init__.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def get_object_data():
     rp = rospkg.RosPack()
     fname = osp.join(rp.get_path(PKG), 'data/object_data_2016.yaml')
     data = yaml.load(open(fname))
+    data = sorted(data, key=lambda d: d['name'])
```
Comment (wkentaro) [inline jsk_apc2016_common/python/jsk_apc2016_common/__init__.py]: oops, I mean sorted in yaml file not in the function.

**Comment by knorth55** [comment]
done

**Comment by wkentaro** [comment]
perfect +1

**Comment by wkentaro** [comment]
```
+ catkin_test_results build

jsk_apc2016_common/test_results/jsk_apc2016_common/roslint-jsk_apc2016_common.xml: 1 tests, 0 errors, 1 failures

Summary: 20 tests, 0 errors, 1 failures
```
linter fails. Please check style of code
```
catkin build --no-deps -iv jsk_apc2016_common --catkin-make-args roslint
```

**Pull request merged**